packages/web-components/src/components/cbp-banner/cbp-banner.scss

```diff
@@ -0,0 +1,31 @@
+
+:root {
+  --cbp-banner-color-bg: var(--cbp-color-info-dark);
+  --cbp-banner-color-fg: var(--cbp-color-text-lightest)
+}
+cbp-banner {
+  display: flex;
+  flex-direction: row;
+
+  &[aria-color="infoBanner"]{
```

Comment:
This should be updated to the reflected property like we discussed. aria-color is not a thing. It could be omitted and put in the core `cbp-banner` selector as well since we don't actually have color variants yet.